BrowserKit/Sources/WebEngine/EngineSessionBackForwardListItem.swift

```diff
@@ -0,0 +1,15 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+import Foundation
+import WebKit
+
+public protocol EngineSessionBackForwardListItem {
+    var url: URL { get }
+
+    var title: String? { get }
+
+    var initialURL: URL { get }
```

Comment:
nit; wondering if we could remove the `initialURL` since this property doesn't seem to be used in Firefox iOS anyway. This would be one less thing the Client has access to for no reasons